Submit/IoTMining/importData.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+# Research: Incremental Learning from IoT for Smart Home Automation
+# Authors: Nguyen Do, Quan Bach
+# Usage:
+# Import Data
+# By running this file, it will pick up datasets in specified in datasets array,
+# import to numpy array with some filters and then partition into weeks
+
+import os
+import utils
+from datetime import datetime, time, timedelta
+from utils import sensorFilter
+import numpy as np
+
+datasets = ["./dataset/data"]
```

Comment:
should we change this to ./dataset/datatest

Reply:
I alread changed the datatest in the dataset folder to data